### ISSUE TITLE
Life savers and deaths

### ISSUE THREAD

**Issue by addicted-to-hot-octolings** [DankMemer/dank-memer-suggestions-and-bugs#155]
**Is your feature request related to a problem? Please describe.**
When I die, and I don't know whether I have a life saver or not. (If you're lazy to check notifs ofc)

**Describe the solution you'd like**
If you would die and you would have a life saver, instead of the original message, it would be: you had a life saver during your death so you haven't lost anything

**Describe alternatives you've considered**
basically, you die with a life saver and instead of the original death message it would type a message about the life saver being consumed

**Comment by HexCoder101** [comment]
That comes in your notifications.
